docs-src/api-overview.md

```diff
@@ -96,6 +96,131 @@ BackendPolicy. For more information on what may be configured with this resource
 in the future, refer to the related [GitHub
 issue](https://github.com/kubernetes-sigs/service-apis/issues/196).
 
+### Route binding
+
+When a Route binds to a Gateway it represents configuration that is applied on
+the Gateway that configures the underlying load balancer or proxy. How and which
+Routes bind to Gateways is controlled by the resources themselves. Route and
+Gateway resources have built-in controls to permit or constrain how they select
+each other. This is useful for enforcing organizational policies for how Routes
+are exposed and on which Gateways. Consider the following example:
+
+> A Kubernetes cluster admin has deployed a Gateway “shared-gw” in the “Infra”
+> Namespace to be used by different application teams for exposing their
+> applications outside the cluster. Teams A and B (in Namespaces “A” and “B”
+> respectively) bind their Routes to this Gateway. They are unaware of each other
+> and as long as their Route rules do not conflict with each other they can
+> continue operating in isolation. Team C has special networking needs (perhaps
+> performance, security, or criticality) and they need a dedicated Gateway to
+> proxy their application to the outside world. Team C deploys their own Gateway 
+> “dedicated-gw”  in the “C” Namespace that can only be used by apps in the "C" 
+> Namespace.
+
+<!-- source: https://docs.google.com/presentation/d/1neBkFDTZ__vRoDXIWvAcxk2Pb7-evdBT6ykw_frf9QQ/edit?usp=sharing -->
+![route binding](images/gateway-route-binding.png)
+
+There is a lot of flexibility in how Routes can bind to Gateways to achieve
+different organizational policies and scopes of responsibility. These are 
+different relationships that Gateways and Routes can have:
+
+- **One-to-one** - A Gateway and Route may be deployed and used by a single 
+  owner and have a one-to-one relationship. Team C is an example of this. 
+- **One-to-many** - A Gateway can have many Routes bound to it that are owned by
+  different teams from across different Namespaces. Teams A and B are an example 
+  of this.
+- **Many-to-one** - Routes can also be bound to more than one Gateway, allowing 
+  a single Route to control application exposure simultaneously across different
+  IPs, load balancers, or networks.
+
+*In summary, Gateways select Routes and Routes control their exposure. When a Gateway selects a Route that allows itself to be exposed, then the Route will bind to the Gateway. When Routes are bound to a
```

Comment:
Nit: The line wrapping is all so perfect in this PR except for here.

Reply:
corrected this, hopefully good to go now!